### PR TITLE
Add eslint, prettier, eslint setting, and start base server

### PR DESCRIPTION
**Pull request by HEKEPOIU**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+console.log(`Server is Fire at http://localhost:${port}`);
+
+app.listen(port, () => {
+    console.log(`Start To listen port ${port}`);
+})
+
+app.get("/", (_req: Request, _res: Response) => {
+    _res.send("Hello This is IFoundApi, checkout /api To see the document.");
+})
```
Comment (sarah20030409) [inline src/index.ts]: res args is used, so delete  '_' .

**Review by sarah20030409 — CHANGES_REQUESTED** [review]
Must add npm i --save-dev @types/node to install package.

File: src/index.ts (hunk 1 above)
Comment (sarah20030409) [reply src/index.ts]: 0d0dd86e62190868e612238387c4263b

Comment (HEKEPOIU) [reply src/index.ts]: resolve

**Review by sarah20030409 — APPROVED** [review]
good

**Pull request merged**